.github/workflows/zowe-cli.yml

```diff
@@ -22,8 +22,10 @@ on:
         required: false
 jobs:
   test:
-
+    if: (github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository) && !contains(github.event.head_commit.message, '[ci skip]')
     runs-on: ${{ matrix.os }}
+    outputs:
+      npm-resolutions: ${{ steps.npm-update.outputs.result }}
```

Comment:
is this a requirement ?
or can the action just set the output without having to explicitly call it out here ?
(the reason why I'm asking is because we might need to add this to the plugins 😋 )

Reply:
The action can set the output on the step level, but in order to set the output on the job level it has to be explicitly called out like this. See https://docs.github.com/en/actions/using-jobs/defining-outputs-for-jobs 🙂